masu/external/date_accessor.py

```diff
@@ -64,20 +67,19 @@ def today(self):
             example: 2018-07-24 15:47:33
 
         """
-        current_date = datetime.today()
+        current_date = datetime.today().replace(tzinfo=pytz.UTC)
```

Comment:
replace doesn't actually change the timezone, so if this is performed with any system timezone other than UTC, it will be inaccurate. 

```
>>> import datetime
>>> import pytz
>>> nowish = datetime.today()
>>> now_utc = datetime.datetime.utcnow()
>>> print(nowish)
2018-10-09 12:47:34.833406
>>> print(now_utc)
2018-10-09 16:47:45.914423
>>> nowish_replaced = nowish.replace(tzinfo=pytz.utc)
>>> print(nowish_replaced)
2018-10-09 12:47:34.833406+00:00
```

For now let's just not touch the `today()` method and just provide a parallel path for the timezone aware values, maybe using separate class attributes for those. It's a crutch, but it gets us through until we address the overarching issue with UTC/timezones.

Reply:
:-1: 

I see no reason adding technical debt is the right answer here, especially when you've already given a better answer - using `utcnow()` instead of `today()`.